### DISCUSSION TITLE
nornir 3.3.0 Perform operations on h3c switches，Result display exception

### DISCUSSION THREAD

**Discussion by wanghuan12345678** [nornir-automation/nornir#848] [Q&A]
**After the script is executed, no operation result is displayed, but operation logs are displayed on the switch**

[root@localhost table-test]# pip freeze |grep nornir
nornir==3.3.0
nornir-napalm==0.4.0
nornir-netmiko==1.0.0
nornir-utils==0.2.0

[root@localhost pystudy]# vim nornir-h3c.py
from nornir import InitNornir
from nornir_napalm.plugins.tasks import napalm_get
from nornir_netmiko import netmiko_send_config,netmiko_save_config,netmiko_send_command
from nornir_utils.plugins.functions import print_result
from nornir.core.filter import F

nr = InitNornir(config_file="config.yaml", dry_run=True)
targets = nr.filter(F(area='SH'))

results = targets.run(netmiko_send_command, command_string='dis clock')
#results = targets.run(task=napalm_get, getters=["get_interfaces_ip"])])

print_result(results

[root@localhost pystudy]# python nornir-h3c.py 
netmiko_send_command************************************************************
* sw3 ** changed : False *******************************************************
vvvv netmiko_send_command ** changed : False vvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvv INFO
^^^^ END netmiko_send_command ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
[root@localhost pystudy]#

**Comment by wanghuan12345678** [comment]
I tried to change the nornir version back to 3.0.0 and it worked

**Comment by ktbyers** [comment]
@wanghuan12345678 Which version of Netmiko are you using in the above two cases (i.e. working and failing case)?

**Reply by wanghuan12345678** [reply]
working:
(nornir) [root@localhost nornir]# pip freeze |grep nornir
nornir==3.0.0
nornir-napalm==0.1.1
nornir-netmiko==0.1.1
nornir-table-inventory==0.4.5
nornir-utils==0.1.1
(nornir) [root@localhost nornir]# pip freeze |grep netmiko
netmiko==3.4.0
nornir-netmiko==0.1.1



failing:
[root@localhost pystudy]# pip freeze |grep nornir
nornir==3.3.0
nornir-napalm==0.4.0
nornir-netmiko==1.0.0
nornir-table-inventory==0.4.5
nornir-utils==0.2.0
[root@localhost pystudy]# pip freeze |grep netmiko
netmiko==4.1.2
nornir-netmiko==1.0.0
